css/selectors/spelling-error.json

```
@@ -23,7 +23,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false,
+              "version_added": "preview",
               "impl_url": "https://webkit.org/b/175784"
```

Same question/suggestion as in the other PR.

```suggestion
              "version_added": "preview"
```